Calibration/HcalCalibAlgos/macros/jobs/isoTrackAlCaCommissioning_cfg.py

```diff
@@ -0,0 +1,50 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+process = cms.Process("ANALYSIS",Run3)
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load('Configuration.StandardSequences.Services_cff')
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load("RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+
+#process.GlobalTag=GlobalTag(process.GlobalTag,'130X_dataRun3_Prompt_v3','')
+process.GlobalTag=GlobalTag(process.GlobalTag,'140X_dataRun3_Prompt_v2' ,'')
+
+process.MessageLogger.cerr.FwkReport.reportEvery = cms.untracked.int32(1000)
+
+if 'MessageLogger' in process.__dict__:
+    process.MessageLogger.HcalIsoTrackX=dict()
+    process.MessageLogger.HcalIsoTrack=dict()
+
+process.options = cms.untracked.PSet( wantSummary = cms.untracked.bool(True) )
+
+process.load('RecoLocalCalo.CaloTowersCreator.calotowermaker_cfi')
+process.towerMakerAll = process.calotowermaker.clone()
+process.towerMakerAll.hbheInput = cms.InputTag("hbhereco")
+process.towerMakerAll.hoInput = cms.InputTag("none")
+process.towerMakerAll.hfInput = cms.InputTag("none")
+process.towerMakerAll.ecalInputs = cms.VInputTag(cms.InputTag("ecalRecHit","EcalRecHitsEB"), cms.InputTag("ecalRecHit","EcalRecHitsEE"))
+process.towerMakerAll.AllowMissingInputs = True
```

Comment:
```suggestion
process.towerMakerAll.hbheInput = "hbhereco"
process.towerMakerAll.hoInput = "none"
process.towerMakerAll.hfInput = "none"
process.towerMakerAll.ecalInputs = [cms.InputTag("ecalRecHit","EcalRecHitsEB"), cms.InputTag("ecalRecHit","EcalRecHitsEE")]
process.towerMakerAll.AllowMissingInputs = True
```
or even (including only what modified wrt the original `calotowermaker`)
```suggestion
process.towerMakerAll.hoInput = "none"
process.towerMakerAll.hfInput = "none"
process.towerMakerAll.AllowMissingInputs = True
```